packages/quill/package.json

```diff
@@ -40,6 +40,8 @@
     "jsdom": "^22.1.0",
     "mini-css-extract-plugin": "^2.7.6",
     "prettier": "^3.0.3",
+    "sass": "^1.71.1",
+    "sass-loader": "^14.1.1",
     "style-loader": "^3.3.3",
     "stylus": "^0.62.0",
```

Comment:
We should remove stylus related packages if they are not used.

Reply:
removed